scratches/test_of_qr.py

```diff
@@ -0,0 +1,94 @@
+import telebot  # importing Telegram API classes and functions
+import libraries.qr_make as qr  # import qr_make lib with qr functions
+import os
+
+# defining global vars
+TOKEN = '2055598237:AAHNAv9osQkVCNqDzbgwclOHMAHY-F01Cb8'
+# list with ids of users, who pressed start button
```

Comment:
Let's keep configs in a separate file (we discussed it in this issue https://github.com/Galaxy13/telegram_bot-Docker.test/issues/1). Moreover, it is not a good practice to document your code using comments. The main drawback is we need to keep in sync code and comments, which is an easy thing to forget. For instance, if you change `known_users` to dict  you will have to update comment.

Reply:
Updated this with adding token.txt